src/Commands/BuildToolsCommand.php

```diff
@@ -39,6 +43,460 @@ public function __construct()
+        $this->setCircleEnvironmentVars($circle_url, $circle_token, $circle_env);
+        $this->addPrivateKeyToCircleProject($circle_url, $circle_token, $privateKey);
+    }
+
+    /**
+     * Provide some simple aliases to reduce typing when selecting common repositories
+     */
+    protected function expandSourceAliases($source)
+    {
+        $aliases = [
+            'example-drops-8-composer' => ['d8', 'drops-8'],
+            'example-drops-7-composer' => ['d7', 'drops-7'],
+            // 'example-wordpress-composer' => ['wp', 'wordpress'],
+        ];
+
+        $map = [strtolower($source) => $source];
+        foreach ($aliases as $full => $shortcuts) {
+            foreach ($shortcuts as $alias) {
+                $map[$alias] = $full;
+            }
+        }
+
+        return $map[strtolower($source)];
+    }
+
+    /**
+     * Detect the upstream to use based on the contents of the source repository.
+     * Upstream is irrelevant, save for the fact that this is the only way to
+     * set the framework on Pantheon at the moment.
+     */
+    protected function autodetectUpstream($siteDir)
+    {
+        // or 'Drupal 7' or 'WordPress'
+        return 'Drupal 8';
+    }
+
+    protected function createSshKeyPair($ssh_key_email, $prefix = 'id')
+    {
+        $tmpkeydir = $this->tempdir('ssh-keys');
+
+        $privateKey = "$tmpkeydir/$prefix";
+        $publicKey = "$privateKey.pub";
+
+        $this->passthru("ssh-keygen -t rsa -b 4096 -f $privateKey -N '' -C '$ssh_key_email'");
+
+        return [$publicKey, $privateKey];
+    }
+
+    protected function cloneExistingGitHub($target, $github_org, $github_token)
+    {
+        $target_org = $github_org;
+        if (empty($github_org)) {
+            $userData = $this->curlGitHub('user', [], $github_token);
+            $target_org = $userData['login'];
+        }
+        $target_project = "$target_org/$target";
+
+        $tmpsitedir = $this->tempdir('local-site');
+        $local_site_path = "$tmpsitedir/$target:dev-master";
+
+        $this->passthru("composer create-project $target_project $local_site_path -n --keep-vcs --stability dev");
+
+        return [$target_project, $local_site_path];
+    }
+
+    protected function createGitHub($source, $target, $github_org, $github_token, $stability = '')
+    {
+        // We need a different URL here if $github_org is an org; if no
+        // org is provided, then we use a simpler URL to create a repository
+        // owned by the currently-authenitcated user.
+        $createRepoUrl = "orgs/$github_org/repos";
+        $target_org = $github_org;
+        if (empty($github_org)) {
+            $createRepoUrl = 'user/repos';
+            $userData = $this->curlGitHub('user', [], $github_token);
+            $target_org = $userData['login'];
+        }
+        $target_project = "$target_org/$target";
+        $remote_url = "git@github.com:${target_project}.git";
+
+        $branch = ''; // TODO: allow non-master here?
```

Comment:
I think that would be `pantheon-systems/example-drops-8-composer/dev-lightning-fist-2`.